src/components/side_panel/chart/line_bar_pie_panel/design_panel.ts

```diff
@@ -11,13 +12,21 @@ import { ChartTitle } from "../building_blocks/title/title";
 
 interface Props {
   figureId: UID;
-  definition: LineChartDefinition | BarChartDefinition | PieChartDefinition;
+  definition:
```

Comment:
Maybe we can defined a main type like 
```js
const type AllChartDefinition = LineChartDefinition | BarChartDefinition | ...
```
to reuse it in the following lines and avoid having to change each line at hand each time ?

Reply:
Not sure if it's a better idea, but we can also use something like `ChartWithAxisDefinition` ?